scripts/terraform/main.tf

```diff
@@ -66,29 +64,41 @@ resource "aws_iam_role_policy_attachment" "github_actions" {
   policy_arn = "arn:aws:iam::aws:policy/AdministratorAccess"
 }
 
-data "aws_caller_identity" "current" {}
+################################################################################
+# Terraform remote state S3 bucket
+################################################################################
+
+module "terraform_state_s3_bucket" {
```

Comment:
S3 bucket module to ensure we have blocked public access, AES256 encryption at least, versioning (key for remote state), etc.

Reply:
I left this out as I was trying to knock it out quickly :)